release/pipeline/dags/istio_common_dag.py

```diff
@@ -227,11 +227,13 @@ def GenerateTestArgs(**kwargs):
       task_id='run_cloud_builder', bash_command=build_template, dag=common_dag)
 
   test_command = """
-    chmod +x /home/airflow/gcs/data/githubctl
```

Comment:
this chmod not working with latest release of airflow, works on /tmp/githubctl

Reply:
I've never seen/tried using chmod without specifying ugoa.  The manpage says blank means all users (a).  Did you try using "u+x"?  Since this is posix I'm guessing there's a decent chance the permissions on /home/airflow/gcs/data don't allow any kind of permission change.  Regardless, it seems like a safer thing to use regardless.  I'm also used to working on products that are paranoid about using /tmp out of concern it's a shared directory that can be accessed and written to by others.  Is there reason to be concerned that the other 3 instances of chmod might have a similar issue?

Reply:
we created a new airflow environment and something seems to have changed. all the other chmod are on current directory which is a temporary directory created by airflow and cleaned up after that step. So I will change to copy in current directory and do a chmod +x